lib/Dialect/FIRRTL/Transforms/RemoveInvalid.cpp

```diff
@@ -121,10 +125,29 @@ void RemoveResetsPass::runOnOperation() {
     }
   }
 
+  for (auto inv : llvm::make_early_inc_range(
+           getOperation().getBody()->getOps<InvalidValueOp>())) {
+    ImplicitLocOpBuilder builder(inv.getLoc(), inv);
```

Comment:
Maybe test if the invalid has no uses to avoid creating a dead value.

Reply:
Added. Great idea.